doc/nep1.rst

```diff
@@ -280,3 +280,6 @@ Conventions for multi-line statements and expressions
   .. code-block:: nim
     startProcess(nimExecutable, currentDirectory, compilerArguments
                  environment, processOptions)
+
+- Use `a..b` instead of `a .. b`, except when `b` contains an operator, for example `a .. -3`.
+  Likewise with `a..<b`.
```

Comment:
```suggestion
  Likewise with `a..<b` and `a..^b`.
```

Is this right?

Reply:
`a..^b` is actually `a ..^ b`, not `a .. ^b`; I've changed the wording